.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - pip3 install .
 
 before_script:
+  - sudo apt-get -y install pandoc
```

Comment:
Thanks for adding. It should be placed under a 'local' `before_script` though, because now the macOS also tries to do `sudo apt-get ...` which it can't, see https://travis-ci.com/github/ComPWA/expertsystem/jobs/340914168. So this line should be moved to line 31:
https://github.com/ComPWA/expertsystem/blob/3460a26c19b16a1089009d4aa04bc6ebde1d4c8b/.travis.yml#L31
This will also fix https://travis-ci.com/github/ComPWA/expertsystem/jobs/340914164